src/plots/plots.js

```diff
@@ -559,6 +562,41 @@ function relinkPrivateKeys(toContainer, fromContainer) {
     }
 }
 
+plots.linkSubplots = function(newFullData, newFullLayout, oldFullData, oldFullLayout) {
```

Comment:
This routine here should make `fullLayout._plots[' '].x()` and `fullLayout._plots[' '].y()` method obsolete.